src/styles/box/header.scss

```diff
@@ -0,0 +1,102 @@
+.header {
+  position: relative;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 6px;
+    left: 0;
+    right: 0;
+    height: 2px;
+    background-color: #fff;
+    box-shadow: 0 2px 4px 0 #0000000d;
+    z-index: -1;
+  }
+
+  &__logo {
+    height: 60px;
+    display: flex;
+    align-items: center;
+    margin-left: 50px;
+    padding-top: 2px;
+
+    &-img {
+      height: 40px;
+      margin: 0;
+    }
+  }
+
+  &__nav {
+    height: 100%;
+    display: flex;
+    align-items: center;
+    margin-right: 50px;
+
+    &-list {
+      display: flex;
+      list-style: none;
+      align-items: center;
+      gap: 20px;
+      padding: 0;
+    }
+
+    &-item {
+      height: 60px;
+      display: flex;
+      align-items: center;
+
+      &:first-child {
+        margin-left: 0;
+        margin-top: 0;
+      }
+
+      &:last-child {
+        margin-right: 0;
+      }
+    }
+
+    &-link {
+      height: 100%;
+      display: flex;
+      align-items: center;
+      color: #060b35;
+      text-decoration: none;
+      font-family: Roboto, sans-serif;
+      font-weight: 500;
+      font-size: 12px;
+
+      &:hover {
+        color: #00acdc;
+      }
+
+      &.is-active {
+        color: #00acdc;
+        text-transform: uppercase;
+        position: relative;
+
+        &::after {
+          content: '';
+          display: block;
+          width: 37px;
+          height: 4px;
+          background-color: #00acdc;
+          border-radius: 8px;
+          position: absolute;
+          bottom: 0;
+          left: 50%;
+          transform: translateX(-50%);
+        }
+      }
+    }
+  }
+
+  @media (min-width: 1200px) {
+    width: 100%;
+  }
```

Comment:
The media query setting `width: 100%;` is redundant because the width is already set to 100% outside of the media query on line 3. Consider removing this media query to keep the code clean.